playwright.config.ts

```diff
@@ -12,7 +12,7 @@ const config: PlaywrightTestConfig = {
   },
   fullyParallel: true,
   forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 2 : 1,
```

Comment:
suggestion: we prefer reverting this change. locally, developers should pay attention if tests seem flaky. Flaky tests deteriorate dev experience in the long run and we shouldnt brush it off

Reply:
Resolved with https://github.com/cpvalente/ontime/pull/1438/commits/c50b08ae3b4e5a05fa0f7f446d5cd0236c788691